config/webpack/webpack.config.js

```diff
@@ -137,6 +138,9 @@ const webpackConfig = (options, env, argv) => {
       new webpack.DefinePlugin({
         CERNER_BUILD_TIMESTAMP: JSON.stringify(new Date(Date.now()).toISOString()),
       }),
+      new webpack.DefinePlugin({
+        SUPPORTED_LOCALES: JSON.stringify(supportedLocales),
```

Comment:
Who is this consumed since this is converted to a string?

Reply:
It can be used as an array if it is defined. Here's a little more information about how the define plugin works. 

https://webpack.js.org/plugins/define-plugin/

Here's a little test I did locally.

```
  console.log(SUPPORTED_LOCALES); // Prints an array of locales if not undefined

  if (SUPPORTED_LOCALES) {
    SUPPORTED_LOCALES.forEach((locale) => {
      console.log(locale);
    });
  }
```